contracts/wrapper/NameWrapper.sol

```diff
@@ -675,23 +654,47 @@ contract NameWrapper is
         emit NameWrapped(node, name, wrappedOwner, fuses);
     }
 
+    function _addLabelAndWrap(
+        bytes32 parentNode,
+        bytes32 node,
+        string memory label,
+        address newOwner,
+        uint96 _fuses
+    ) internal {
+        bytes memory name = _addLabel(label, names[parentNode]);
+        _wrap(node, name, newOwner, _fuses);
+    }
+
+    function _transferAndBurnFuses(
+        bytes32 node,
+        address newOwner,
+        uint96 _fuses
+    ) internal {
+        (address owner, ) = getData(uint256(node));
+        _transfer(owner, newOwner, uint256(node), 1, "");
```

Comment:
Probably worth adding a conditional here so it doesn't do this if `owner == newOwner`.